about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link
+  rel="stylesheet"
+  href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+  integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+  crossorigin="anonymous"
+/>
+<link rel="stylesheet" href="css/styles.css">
+  <title>Zambia all basketball show</title>
+</head>
+<body>
+  <header class="head">
+    <nav class="navigation">
+      <a id="bar" class="hum" href="#">
+        <img src="images/menu-trigger-1.png" alt="humberger image">
```

Comment:
- [ ] The background of the hamburger image should be transparent, please delete the white space on top of the document:
<img width="426" alt="Screenshot 2022-04-14 at 18 11 29" src="https://user-images.githubusercontent.com/77038610/163408715-986ddb64-bddf-4bf2-bdcf-a353495559bf.png">

Reply:
- [ ] [REQUIRED] As required by the previous reviewer, you have done a great implementation but the UI still does not enhance good accessibility yet.  Currently, the hamburger is not visible due to the color combination of the background and the `h1` tag.
To fix this: Kindly ensure there is a margin-top on the `h1` tag and reduce the opacity of the background image as it looks too solid/high.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-14 22-01-50](https://user-images.githubusercontent.com/58692590/163476752-71758dcd-04dc-4ff1-b980-10078890f609.png) | ![Screenshot from 2022-04-14 22-03-20](https://user-images.githubusercontent.com/58692590/163476802-b878b1a2-6a93-4c98-a1b7-4f68d659fdbe.png)